src/error_logger_lager_h.erl

```diff
@@ -74,7 +74,7 @@ set_high_water(N) ->
     gen_event:call(error_logger, ?MODULE, {set_high_water, N}, infinity).
 
 -spec init(any()) -> {ok, #state{}}.
-init([HighWaterMark]) ->
+init([{sink, Sink}, HighWaterMark]) ->
```

Comment:
Sink is only necessary for the throttle backend, so all of the other backends will ignore it. Not sure why I didn't name the binding appropriately there.